### DISCUSSION TITLE
Issue Building APK for Release 6.0

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#708] [General Discussion]
Hello Team,

I am encountering an issue while attempting to build the APK for Release 6.0. After launching the APK, I'm experiencing an error. I followed the process by running the ./build.sh command, and the build was generated successfully. However, upon launching the APK, the issue arises.

Steps Taken:
1. Ran ./build.sh command.
2. Observed successful build.
3. Launched the APK.
4. Encountered error.

Screenshot: This is the issue 
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/3453f2e0-eb94-4e1f-9b7b-62597d561520)

I kindly request assistance in resolving this issue so that I can successfully build and launch the APK for Release 6.0.

cc: @rajeevsathish @mohitga @vinukumar-vs @ashokreddy1208 @swayangjit

**Comment by siddhishinde0723** [comment]
I tried rebuilding the apk after removing all the dependencies but still faced the same issue.

**Reply by swayangjit** [reply]
@siddhishinde0723  For 6.0.0 the branch is https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/tree/release-6.0.0_v13

**Reply by siddhishinde0723** [reply]
@swayangjit i have cloned the same branch and am having issues.

**Reply by siddhishinde0723** [reply]
@swayangjit I tried rebuild after a successful apk build it was the browser application where i got this issue 
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/519099dc-9425-4408-8a80-b0f52d96f6c7)

**Reply by swayangjit** [reply]
@siddhishinde0723 browser application ?

**Reply by siddhishinde0723** [reply]
@swayangjit here is the screen for the above issue 
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/2a8ee7f0-20ca-4da0-b7d0-4703727eebcf)

After apk build success, i am getting the issue

**Reply by swayangjit** [reply]
@siddhishinde0723  Can you check if SharedPreference plugin is installed or not ?

**Reply by siddhishinde0723** [reply]
@swayangjit No it is not installed.
Can you please share that.

**Reply by swayangjit** [reply]
@siddhishinde0723  Its avaliable in package.json. Please do install and let me know.

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

We hope that you were able to fix the issue. If yes, could you please close this thread from your end?

cc: @swayangjit

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
